src/test/datascience/jupyter/serverSelector.unit.test.ts

```diff
@@ -88,206 +107,453 @@ suite('DataScience - Jupyter Server URI Selector', () => {
         disposeAllDisposables(disposables);
     });
 
-    test('Local pick server uri', async () => {
-        const { selector, storage } = createDataScienceObject('', true);
-        await selector.selectJupyterURI();
-        let value = await storage.getUri();
-        assert.equal(value, Settings.JupyterServerLocalLaunch, 'Default should pick local launch');
+    suite('Experimental', () => {
```

Comment:
Two copies of the tests. 

One for new way the server selector works
One for the old way.